src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arrayOfStyle = sourceString.split(';');
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. Try to give your variables more descriptive names that reflect their purpose in the code.